api/client/gro_client.py

```diff
@@ -82,6 +88,48 @@ def get_df(self):
                 self._data_frame = self._data_frame.merge(tmp, how='outer')
         return self._data_frame
 
+    def GDH(self, gdh_selection):
+        """Wrapper for get_data_points with alternative input and output style.
+
+        The selection of data series to retrieve is encoded in a
+        'gdh_seletion' string of the form
+        <metric_id>-<item_id>-<region_id>-<source_id>-<frequency_id>
```

Comment:
This is mentioned in the PRD comments, but for consistency we should probably also always include partner_region in the string even if a partner_region hasn't be explicitly selected.

Reply:
@nemozen also units.

In @Fryolator2 's local example, the string requires <metric_id>-<item_id>-<region_id>-<source_id>-<frequency_id> and then has the options for partner region and units on the end. If these are not specified, they are assumed to be 'world' and the default.

This same functionality, or to always require these two would be important.